src/assets/contributors/names.json

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
do we still need this file if `contributers.json` exists?

Reply:
I committed `names.json` before I knew `contributors.json` existed 😭 . I ended up deleting `names.json`